book/08_Flood_analysis/3_Retrieving_FloodData.md

```diff
@@ -234,13 +236,13 @@ COLORS[253] = (150, 150, 150, 1)
 ```python
 img = dataset.hvplot.quadmesh(title = 'DSWx data for May 2024 Texas floods',
```

Comment:
We're going to have to explain this step in detail in one of the notebooks on software tools.

There may be a way to specify these plot options using the `opts` system built in to HoloViews/GeoViews. Let's discuss this.